app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
@evelynleems and I discussed this at length, and she'll summarise her opinions here in the next comment.